x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/nlp/NerProcessor.java

```diff
@@ -57,11 +62,56 @@ boolean isBeginning() {
         }
     }
 
-
     private final BertRequestBuilder bertRequestBuilder;
+    private final List<String> classificationLabels;
+    private final IobTag[] iobMap;
 
-    NerProcessor(BertTokenizer tokenizer) {
+    NerProcessor(BertTokenizer tokenizer, @Nullable List<String> classificationLabels) {
         this.bertRequestBuilder = new BertRequestBuilder(tokenizer);
+        this.classificationLabels = classificationLabels;
```

Comment:
Should we set this to the default value if null or empty so that we have a single way of dealing with it throughout the rest of the code?

Reply:
Actually `classificationLabels` does not need to be a member variable as they are only used to build the I-O-B tag map so I've removed it. I think that makes things clearer